packages/ai-rules/.ruler/frontend/QUICKSTART.md

```
@@ -0,0 +1,140 @@
+
+## Decision Trees
+
+### Where Do I Put My Code?
+
+```
+I need to add...
+
+├─ New feature with UI?
+│  └─> Create FeatureName/ folder
+│     ├─> FeatureContainer.tsx (data + state)
+│     ├─> FeatureView.tsx (presentation)
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)